commerce_invoice.module

```diff
@@ -324,22 +316,23 @@ function commerce_invoice_load_current($order) {
  * @param int         $limit
  *   Limit the number of invoices to return, or 0 for no limit.
  * @param array       $properties
- *   Properties to filter invoices by.
+ *   Properties to filter invoices by. Each property should be an array of 3
```

Comment:
Looks like the parameter order is wrong in the docblock, could you amend that in this PR too?